cli/medperf/tests/entities/test_cube.py

```diff
@@ -12,8 +12,10 @@
 PATCH_CUBE = "medperf.entities.cube.{}"
 CUBE_PATH = "cube_path"
 PARAMS_PATH = "params_path"
-TARBALL_PATH = "tarball_path"
-TARBALL_HASH = "tarball_hash"
+TARBALL_PATH = "additional_files_tarball_path"
+TARBALL_HASH = "additional_files_tarball_hash"
+IMG_PATH = "image_tarball_url"
+IMG_HASH = "image_tarball_hash"
```

Comment:
Another spot we should eventually clean up to just read "image" instead of "image_tarball"